source/fare/fare.h

```diff
@@ -259,10 +263,16 @@ struct Label {
             return nb_undefined_sub_cost < l.nb_undefined_sub_cost;
         if (cost.value != l.cost.value)
             return cost.value < l.cost.value;
+        if (tickets.size() != l.tickets.size()) {
```

Comment:
:+1: This explains the 1-cent fix for fares, I guess.

Reply:
yep ! After this commit, it should work well with 0-cent ticket